hikari/impl/rest.py

```diff
@@ -549,6 +550,7 @@ def __init__(
         executor: typing.Optional[concurrent.futures.Executor],
         http_settings: config.HTTPSettings,
         max_rate_limit: float,
+        max_retries: int = 5,
```

Comment:
Wouldnt it be better to default at 3 with a top at 5?

Reply:
In the issue, @FasterSpeeding suggested to put 5 retries. But are you sure you want to limit it to 5? Since this feature was about "giving more control to users", they should be able to do as many retries as they want to be honest.